README.md

```diff
@@ -1,2 +1,23 @@
-# HW01
-Edit the readme
+---
+title: "Homework 1"
+author: "Ben Wang"
+---
+
+### bwang5@uchicago.edu
+
+Hello all!
+My name is Ben, I'm an incoming first year student in the **University of Chicago** *Neurobiology* program [link here](https://neuroscience.uchicago.edu/). I am interested in studying the molecular mechanisms behind automatic functions such as breathing, pain and prioprioception. I have no prior experience in coding but have always been interested in learning!
+
+I am originally from Winnipeg, Canada, and came to the US for college. I majored in molecular and cellular biology and minored in music.
+
+Some of my hobbies/interests:
+
+* Piano
+* Ultimate frisbee
+* Traveling and trying to memorize random phrases from obscure local languages
+* Running
+   * Heading along the lakeshore in the mornings, give me a shout if you also like running! 
+
+![View from one of the downtown Winnipeg parks!](https://i.pinimg.com/originals/eb/bf/96/ebbf9686a19a8fadff95b3479e5e5fe6.jpg)
```

Comment:
First this is a beautiful image!  I love the snow and can't wait to be in Chicago during the winter.

Second, it looks like Nabilah, you, and myself all had the same issue with the image caption not displaying.  I really wonder why that is, it seems we all used the same format.

Reply:
I was following your various commits and I am so glad you finally got the image to appear Ben! Although, we still have that caption problem. I haven't got a chance to look into it but I shall let you guys know if I ever found out!